cms/djangoapps/contentstore/tasks.py

```diff
@@ -630,6 +630,9 @@ def create_export_tarball(course_module, course_key, context, status=None):
     Updates the context with any error information if applicable.
     """
     name = course_module.url_name
+    if isinstance(name, six.binary_type):
+        name = name.decode('utf-8')
+
     export_file = NamedTemporaryFile(prefix=name + '.', suffix=".tar.gz")
```

Comment:
Should we instead be passing `prefix` and `suffix` as explicitly bytes, rather than converting them all over to strings?

Reply:
It turns out that the export file name is combined with other strings later and making just this bytes causes other failures when path tries to combine unicode and byte strings.